guides/choosing_mono_or_multi_repo.md

```diff
@@ -0,0 +1,96 @@
+# Choosing between a mono repo and a multi repo
+
+A mono repo is a single repository that contains all the code for multiple projects. A multi repo is a collection of repositories, each containing the code for a single project. Both approaches have their own advantages and disadvantages. This guide will help you decide which approach is best for your project.
+
+Developers should proactively choose between a mono repo and a multi repo based on the project requirements. The decision should be made at the beginning of the project, as it can be difficult to switch between the two later on.
+
+The decision should be reviewed periodically to ensure that it still meets the project requirements. If the project requirements change, it may be necessary to switch between a mono repo and a multi repo.
+
+## Considerations
+
+Teams should consider the following key drivers for choosing between a mono repo and a multi repo:
```

Comment:
I think the key question is how are you going to manage dependencies?

In my mind, monorepos can only really work when you have an integrated team who can potentially all work on all the components/packages.

If you have ownership of different components kind of sub-divided within the team, with truly decoupled services then monorepos probably slow the whole process down.

Reply:
Yes definitely agree.  Don't even get me started on cross team merge conflicts.

I've reiterated this point a bit more throughout the content